pallets/parachain-system/src/lib.rs

```diff
@@ -328,6 +328,7 @@ pub mod pallet {
 					let validation_code = <PendingValidationCode<T>>::take();
 
 					Self::put_parachain_code(&validation_code);
+					<T::OnValidationData as OnValidationData>::on_validation_code_applied();
```

Comment:
Trait and associated type should be renamed.

Reply:
Sorry, I don't get exactly how to change this one. Can you please propose the change to be merged?

Reply:
The associoated type `type OnValidationData` should be renamed, similar to the trait as I have said below.